app/controllers/AccountHomeController.scala

```diff
@@ -75,9 +80,18 @@ class AccountHomeController @Inject() (
   def startApplication: Action[AnyContent] =
     identify.async {
       implicit request =>
-        for {
-          draftId <- counterRepository.nextId(CounterId.DraftId)
-          _       <- userAnswersService.set(UserAnswers(request.userId, DraftId(draftId)))
-        } yield Redirect(navigator.startApplicationRouting(request.affinityGroup, DraftId(draftId)))
+        AuthUserType(request) match {
+          case None           =>
+            // TODO: Update error handling
+            Future.successful(Redirect(controllers.routes.JourneyRecoveryController.onPageLoad()))
```

Comment:
Do we have a "Cannot use this service" page, that we can redirect the user to instead?